arpack-ng-config.cmake.in

```diff
@@ -17,9 +17,43 @@ set(exec_prefix "@exec_prefix@")
 set(libdir "@libdir@")
 set(includedir "@includedir@")
 
+include(CMakeFindDependencyMacro)
+if (NOT TARGET BLAS::BLAS)
+    find_dependency(BLAS REQUIRED)
+    if (NOT TARGET BLAS::BLAS) # Create target "at hand" to ensure compatibility if cmake version < 3.18
+        add_library(BLAS::BLAS INTERFACE IMPORTED)
+        set_target_properties(BLAS::BLAS PROPERTIES INTERFACE_LINK_LIBRARIES "${BLAS_LIBRARIES}")
+    endif()
+endif()
+if (NOT TARGET LAPACK::LAPACK)
+    find_dependency(LAPACK REQUIRED)
+    if (NOT TARGET LAPACK::LAPACK) # Create target "at hand" to ensure compatibility if cmake version < 3.18
+        add_library(LAPACK::LAPACK INTERFACE IMPORTED)
+        set_target_properties(LAPACK::LAPACK PROPERTIES INTERFACE_LINK_LIBRARIES "${LAPACK_LIBRARIES}")
+    endif()
+endif()
+if (@MPI@)
+    if (NOT TARGET MPI::MPI_Fortran) # Search only if not already found by upper CMakeLists.txt
```

Comment:
Missing MPI:MPI_C and MPI::MPI_CXX in case of ICB? https://github.com/opencollab/arpack-ng/blob/645db9e6472283408563e3e2dc99c5deb8272c5d/CMakeLists.txt#L220

Reply:
This has been fixed in a new commit.